lms/djangoapps/grades/api/views.py

```diff
@@ -294,73 +289,70 @@ class GradesBulkAPIView(ListAPIView):
         JwtAuthentication,
         SessionAuthentication
     )
+    serializer_class = GradeBulkAPIViewSerializer
     permission_classes = (IsAuthenticated, IsStaffOrOwner)
 
-    def get(self,  request):
+    def get(self, request):
 
+        return Response("Only POST method allowed.")
```

Comment:
Please change this to gooi a 404. or protect the url definition to only allow for the post. this is a little too open and can allow someone to quickly figure out how to manipulate the api.